src/core_modules/capture-core/components/WidgetRelatedStages/WidgetRelatedStages.component.js

```diff
@@ -28,6 +29,16 @@ const WidgetRelatedStagesPlain = ({
         occurredLabel,
         enrollmentId,
     });
+
+    const { loading: orgUnitLoading, data: orgUnitData } = useDataQuery({
+        orgUnits: {
+            resource: 'me',
+            params: {
+                fields: ['organisationUnits[id,path,displayName,children::isNotEmpty]'],
```

Comment:
I think we might have misunderstood each other here - fetching orgunits through the `/me`-endpoint does not necessarily filter the children array for whether you have access or not. Let's say a user only has access to the `Bo`-orgunit. With this API-call, children would be present (true) as Bo does have a child. However, the user does not have access to this orgUnit. 

When fetching through the `/api/organisationUnits`-endpoint, there are filters to make sure you are searching for all orgunits `withinUserScope` or `withinUserSearchScope`. If this array is bigger than two, that means the user has more than two orgunits assigned.